src/string-class.js

```diff
@@ -0,0 +1,137 @@
+' use strict';
+
+const StringClass = {
+  /**
+   * Tests if a string contains vowels
+   * @return {Boolean} True or false depending on the match
+   */
+  hasVowels() {
+    return /[aeiouAEIOU]/g.test(this);
+  },
+  /**
+   * converts strings to uppercase
+   * @return {String} string that has been converted to uppercase
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g, (character) => {
+      return String.fromCharCode(character.charCodeAt() - 32);
+    });
+  },
+  /**
+   * converts strings to lowercase
+   * @return {String} string that has been converted to lowercase
+   */
+  toLower() {
+    return this.replace(/[A-Z]/g, (character) => {
+      return String.fromCharCode(character.charCodeAt() + 32);
+    });
+  },
+  /**
+   * converts strings to title case
+   * @return {String} string that has been converted to title case
+   */
+  ucFirst() {
+    return `${this.slice(0, 1).toUpper()}${this.slice(1).toLower()}`;
```

Comment:
Does the instruction say all other characters should be converted to lower case?